include/rdma/fi_domain.h

```diff
@@ -115,6 +115,11 @@ struct fid_av {
  * Tracks registered memory regions, primarily for remote access,
  * but also for local access until we can remove that need.
  */
+
+#define FI_HMEM_HOST_ALLOC		(1ULL << 60)
+#define FI_HMEM_DEVICE_ONLY		(1ULL << 61)
```

Comment:
We can't change these flags.  This breaks the API.  I was only saying that they *should* have been given a different value, not that can be.

Reply:
phew.. near miss. I will revert this part. Thanks!

But I doubt if this will actually break anything at this point. Did a quick code search, and the usage is strictly local, i.e. no exchange between endpoints. So I guess their values can be changed.... later.

Reply:
These have been exposed to existing applications, which could be using them as input into a memory registration call.  E.g. fabtests uses them.  So, they can't change even later.

Reply:
😬